mitodl/mitxonline
# 5999 provide standardized time commitment values in the mitx online api

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/5999

### Description (What does it do?)
Adds min and max weekly hours to the courses V2 API. 

### How can this be tested?

1. Edit a course CMS page
2. Enter in a number for the max and min hours per week.
3. Visit http://mitxonline.odl.local:8013/api/v2/courses/ and verify that the max and min hours per week are shown.
